static/js/constants.js

```diff
@@ -110,6 +110,10 @@ export const HIGHEST_EDUCATION_CHOICES = [
 export const ALERT_TYPE_TEXT = "text"
 export const ALERT_TYPE_UNUSED_COUPON = "unused-coupon"
 
+// Error message for failed API calls
+export const GENERIC_ERROR_MESSAGE =
+  "Sorry, we don't know what's happening. Please try again or contact support if the problem persists."
```

Comment:
Please use the error message I specified in https://github.com/mitodl/mitxpro/issues/843#issuecomment-531338936